### PR TITLE
[CartBundle] CS fixes and missing specs

### PR DESCRIPTION
**Pull request by aRn0D**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | refactoring 
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | no
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Comment by aRn0D** [comment]
@pjedrzejewski  Do I revert my last commit or I try to make it work?

**Comment by aRn0D** [comment]
@pjedrzejewski At midday, I will revert my last commit. I will focus on simple bundle configuration and specs.

**Comment by aRn0D** [comment]
And I will fix array syntax !

**Comment by aRn0D** [comment]
@pjedrzejewski green !

File: src/Sylius/Bundle/CartBundle/spec/Doctrine/ORM/CartRepositorySpec.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        $builder->from(Argument::any(), 'o')->shouldBeCalled()->willReturn($builder);
+        $builder->leftJoin('o.items', 'item')->shouldBeCalled()->willReturn($builder);
+        $builder->addSelect('item')->shouldBeCalled()->willReturn($builder);
+        $builder->andWhere(Argument::any())->shouldBeCalled()->willReturn($builder);
+        $builder->andWhere(Argument::any())->shouldBeCalled()->willReturn($builder);
+        $builder->setParameter('now', Argument::type('\DateTime'))->shouldBeCalled()->willReturn($builder);
+        $builder->setParameter('state', OrderInterface::STATE_CART)->shouldBeCalled()->willReturn($builder);
+
+        $builder->getQuery()->shouldBeCalled()->willReturn($query);
+        $query->getResult()->shouldBeCalled();
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CartBundle/spec/Doctrine/ORM/CartRepositorySpec.php]: ``` ->willReturn(array($cart1)); ```

File: src/Sylius/Bundle/CartBundle/spec/Doctrine/ORM/CartRepositorySpec.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+        $this->findExpiredCarts();
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CartBundle/spec/Doctrine/ORM/CartRepositorySpec.php]: ``` ->shouldReturn(array($cart1)); ```

**Comment by aRn0D** [comment]
@michalmarcinkowski I updated the PR, the previous build failed for PHP5.3 but I don't know (it was not a problem about array syntax)

**Comment by aRn0D** [comment]
ping @pjedrzejewski this is green :p

**Comment by michalmarcinkowski** [comment]
:+1:

**Comment by aRn0D** [comment]
ping @pjedrzejewski

**Comment by aRn0D** [comment]
ping @pjedrzejewski

**Comment by pjedrzejewski** [comment]
Thank you Arnaud! :+1:

**Pull request merged**